contracts/land/LANDRegistry.sol

```diff
@@ -294,7 +294,7 @@ contract LANDRegistry is Storage, Ownable, FullAssetRegistry, ILANDRegistry {
     }
   }
 
-  function setUpdateOperator(uint256 assetId, address operator) external onlyOwnerOf(assetId) {
+  function setUpdateOperator(uint256 assetId, address operator) external onlyAuthorized(assetId) {
```

Comment:
Should this also modify [contracts/land/LANDRegistry.sol:setUpdateOperator()](https://github.com/decentraland/land/blob/master/contracts/land/LANDRegistry.sol#L297)?

Reply:
@josephholsten What do you mean?